akka-http-core/src/main/mima-filters/10.1.x.backwards.excludes/query-string-encoding.backwards.excludes

```diff
@@ -0,0 +1 @@
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.http.impl.model.parser.CharacterClasses.raw-query-char")
```

Comment:
Maybe it needs name mangling and mima is reporting the unmangled names? I don't think `-` is allowed in symbol names.

Reply:
the strange thing is it succeeds for me locally...

Reply:
figured it out: we're duplicating some of the mima code in the pullrequestValidation task, and the '10.1.x' support wasn't in there yet.

Reply:
Good catch